src/client/transaction.go

```diff
@@ -335,3 +335,311 @@ func (c *ppsBuilderClient) CreatePipeline(ctx context.Context, req *pps.CreatePi
 	c.tb.requests = append(c.tb.requests, &transaction.TransactionRequest{CreatePipeline: req})
 	return nil, nil
 }
+
+func (c *pfsBuilderClient) ActivateAuth(ctx context.Context, req *pfs.ActivateAuthRequest, opts ...grpc.CallOption) (*pfs.ActivateAuthResponse, error) {
```

Comment:
@robert-uhl is this generated? Could you elaborate on what's going on here and how it relates to the other changes?

Reply:
I believe that I picked the wrong leg of a diff when merging.

I’ve redone the merge.